docs/source/user-guide/vision/quickstarts/plot_classification_tutorial.py

```diff
@@ -191,76 +146,98 @@ def imshow(inp, title=None):
 print("Image values", batch[1][0])
 
 #%%
-# Implementing the ClassificationData class
+# Implementing the VisionData class
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# The first step is to implement a class that enables deepchecks to interact with your model and data.
-# The appropriate class to implement should be selected according to you models task type. In this tutorial,
-# we will implement the classification task type by implementing a class that inherits from the
-# :class:`deepchecks.vision.classification_data.ClassificationData` class.
-
-# The goal of this class is to make sure the outputs of the model and of the dataloader are in the correct format.
+# The checks in the package validate the model & data by calculating various quantities over the data, labels and
+# predictions. In order to do that, those must be in a pre-defined format, according to the task type.
+# In the following example we're using pytorch. To see an implementation of this in tensorflow, please refer to #TODO
+# For pytorch, we will use our DataLoader, but we'll create a new collate function for it, that transforms the batch to
+# the correct format. Then, we'll create a VisionData object, that will hold the data loader.#
+#
+# The VisionData class contains additional data and general methods intended for easy access to relevant metadata
+# for object detection ML models validation.
 # To learn more about the expected format please visit the API reference for the
-# :class:`deepchecks.vision.classification_data.ClassificationData` class.
-
-class AntsBeesData(ClassificationData):
-
-    def __init__(self, *args, **kwargs):
-      super().__init__(*args, **kwargs)
+# :doc:Deepchecks' format </user-guide/vision/supported_tasks_and_formats>``
+#
+# First, we'll create the collate function that will be used by the DataLoader.
+# In pytorch, the collate function is used to transform the output batch to any custom format, and we'll use that
+# in order to transform the batch to the correct format for the checks.
+
+def deepchecks_collate_fn(batch):
+    """Return a batch of images, labels and predictions for a batch of data. The expected format is a dictionary with
+    the following keys: 'images', 'labels' and 'predictions', each value is in the predefined format for the task:
+
+    For images, The expected format is an iterable of images,
+    where each image is a numpy array of shape (height, width, channels). The numbers in the array should be in the
+    range [0, 255]
+
+    For labels, The expected format is an array of shape (N,), where N is the number of samples. Each element is an
+    integer representing the class index.
+
+    For predictions, The expected format is an array of shape (N, n_classes), The expected format is an array of
+    shape (N, n_classes), where N is the number of samples. Each element is an array of length n_classes that represent
+    the probability of each class.
```

Comment:
would drop all of this and replace with link to https://docs.deepchecks.com/stable/user-guide/vision/supported_tasks_and_formats.html